fakefs.gemspec

```diff
@@ -9,14 +6,11 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Chris Wanstrath", "Scott Taylor", "Jeff Hodges", "Pat Nakajima", "Brian Donovan"]
   spec.email         = ["chris@ozmm.org"]
   spec.description   = %q{A fake filesystem. Use it in your tests.}
-  spec.summary       = %q{A fake filesystem. Use it in your tests.}
-  spec.homepage      = "http://github.com/defunkt/fakefs"
+  spec.summary       = spec.description
+  spec.homepage      = "https://github.com/defunkt/fakefs"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files`.split($/)
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-  spec.require_paths = ["lib"]
+  spec.files         = `git ls-files lib README.markdown LICENSE`.split($/)
```

Comment:
- all users will directly benefit from the smaller payload + smaller git size when vendored
 - Installing gems via packages is weird / uncommon -> very little impact
 - Most test suites do not run without proper bundling/setup so maintainers most likely cannot run them anyway
 - Lots of popular libraries do not sip test files -> if rails does not do it, we don't need to either